### PR TITLE
Fix an add_filter call

### PR DESCRIPTION
**Pull request by szepeviktor**
Discovered by **PHPStan**

@jeherve And few other improvements.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
   "support": {
     "issues": "https://github.com/jeherve/nostr-verify/issues"
   },
-  "require": {},
+  "require": {
+    "php": ">=7.2"
```
Comment (szepeviktor) [inline composer.json:11]: https://github.com/jeherve/nostr-verify/blob/ff3f1a28672dffd8b4c815dfacae0b1e8506fa3f/readme.txt#L5

File: nostr-verify.php [hunk 2]
```diff
@@ -96,7 +96,7 @@ function ( $vars ) {
 /**
  * Parse the nostr request and render the document.
  *
- * @param WP $wp WordPress request context.
+ * @param \WP $wp WordPress request context.
```
Comment (szepeviktor) [inline nostr-verify.php:99]: Docblocks need FQCN-s.

**Review by jeherve — APPROVED** [review]
Thanks for the contribution!

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**